README.md

```
@@ -1592,22 +1536,19 @@ npm run build
 ## 📄 License
 This project is licensed under the **MIT License** - see the [LICENSE](./LICENSE) file for full details.
 
-```
+```text
 MIT License - feel free to use this in your projects!
 ```
 
 ## 💬 Final Thoughts
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)